mock-infrastructure/src/main/java/com/sequenceiq/mock/spi/MockResponse.java

```diff
@@ -16,6 +18,8 @@ public class MockResponse {
 
     private String clss;
 
+    private Set<String> bodyContains;
```

Comment:
Although I had a guess that what this field is used for it took a minute to make sure in the code. Can you please use a more descriptive name like `requestMatchers`? This would make it clear that this mock response will be returned if all parts matches in the request.